build.project

```diff
@@ -69,6 +69,7 @@ b_build() {
 #
 b_image() {
 	b_image_target
+	declare -F b_image_project && b_image_project
```

Comment:
this returns 1 which makes build.sh exit with return code 1

Reply:
Ok. In the next version I'l try something like this:
```suggestion
	declare -F b_image_project && b_image_project
	return 0
```